### DISCUSSION TITLE
BLAST tabular output

### DISCUSSION THREAD

**Discussion by emc2rae** [bbuchfink/diamond#539] [Q&A]
Hi! I'm having trouble getting the output I need. The "sseqid" option seems to behave differently depending on what other options are included. Is there any way to get both the ID number AND the accession in the same output (in any format)?

--outfmt 6 sseqid evalue bitscore
1334999579     0.0     1567

--outfmt 6 sseqid stitle  evalue bitscore
WP_103141275.1	WP_103141275.1 DNA gyrase subunit A [Bacillus cereus]	0.0	1567

**Comment by bbuchfink** [comment]
Hi, can you tell me what diamond version and what database format you are using?

**Comment by emc2rae** [comment]
Hi! I'm using diamond v2.0.13.151 and version 4 of the Blast database

**Reply by bbuchfink** [reply]
Are you using a self-made blast database? If so, you need to use `-parse_seqids` with `makeblastdb` to get the accessions. Getting both the accession and the number is not supported at the moment.
